layouts/shortcodes/supported-platforms.html

```diff
@@ -0,0 +1,32 @@
+<!-- Generated using the data in data/platforms.yaml -->
+{{ $platforms := site.Data.platforms }}
+<table>
+  <thead>
+    <tr>
+      <th>
+        Language
+      </th>
+      <th>
+        Platform
+      </th>
+      <th>
+        Compilers
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    {{- range $platforms -}}
```

Comment:
Nit: I'm not a Hugo expert, but if this were Jekyll, I'd instead write `{{- range $platforms }}`, dropping the `-` at the end. Is this supported by Hugo?

Reply:
In this case, the `{{- ... -}}` ensures that this template doesn't take up space, and thus preserves the table formatting.

Reply:
Right, as is the case for the Jekyll equivalent. I'll try to find the relevant section in the Hugo docs to see if there are variants of that directive.

Reply:
Fixed. But in general this does not make a difference.